Makefile

```diff
@@ -28,10 +28,11 @@ GCP_PROJECT ?= k8s-skaffold
 GKE_CLUSTER_NAME ?= integration-tests
 GKE_ZONE ?= us-central1-a
 
-SUPPORTED_PLATFORMS := linux-$(GOARCH) darwin-$(GOARCH) windows-$(GOARCH).exe
+SUPPORTED_PLATFORMS = linux-$(GOARCH) darwin-$(GOARCH) windows-$(GOARCH).exe
```

Comment:
After going through http://www.gnu.org/software/make/manual/make.html#Flavors, 
I feel simply expanded variables and make more sense in this Makefile. 

Is there a reason why you changed to "recursively expanded" ?